File: packages/tasks-gen/snippets-fixtures/text-to-video/js/huggingface.js/0.replicate.js
import { InferenceClient } from "@huggingface/inference";

const client = new InferenceClient(process.env.HF_TOKEN);

const video = await client.textToVideo({
    provider: "replicate",
    model: "tencent/HunyuanVideo",
	inputs: "A young man walking on the street",
});
// Use the generated video (it's a Blob)

File: packages/tasks-gen/snippets-fixtures/text-to-video/python/huggingface_hub/0.replicate.py
from huggingface_hub import InferenceClient, get_token

client = InferenceClient(
    provider="replicate",
    api_key=get_token(),
)

video = client.text_to_video(
    "A young man walking on the street",
    model="tencent/HunyuanVideo",
)